include/avif/avif.h

```diff
@@ -7,23 +7,46 @@
 #include <stddef.h>
 #include <stdint.h>
 
-#ifndef AVIF_API
-#if defined(AVIF_BUILDING_SHARED_LIBS)
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// ---------------------------------------------------------------------------
+// Export macros
+
+// AVIF_BUILDING_SHARED_LIBS should only be defined when libavif is being built
+// as a shared library.
+// AVIF_DLL should be defined if libavif is a shared library. If you are using
+// libavif as CMake dependency, through CMake package config file or
+// through pkg-config, this is defined automatically.
+//
+// Here's what `AVIF_API` will be defined to in shared build:
```

Comment:
Nit: Change "defined to" to "defined as" (or perhaps "defined to be") here and in line 28.

Also, I think AVIF_API does not need to be quoted in ``. Is `` part of the markdown language?

Reply:
Updated. Thanks for your advices on my expressions.

Reply:
You're welcome!

I just used GitHub's "Squash and merge" UI to merge this pull request. It did not seem to give me an opportunity to edit the commit message, so the commit message is just the summary lines of the four commits. For your other pull request I will try the command-line instructions.

Reply:
Actually, the "Squash and merge" UI has a dialog box but I missed it. I will try it now with the other pull request.